### ISSUE TITLE
Bug (a11y): Blogs by Author page is missing title in the html head

### ISSUE THREAD

**Issue by lsr-explore** [accessiblecommunity/accessiblecommunity.github.io#554]
# Blogs by Author page is missing title in the html head

https://accessiblecommunity.org/blog/authors/

The tab displays the url instead of Blogs by author

```
+     "description": "Ensure each HTML document contains a non-empty <title> element",
+     "help": "Documents must have <title> element to aid in navigation",
+     "helpUrl": "https://dequeuniversity.com/rules/axe/4.11/document-title?application=playwright",
```

<img width="2006" height="1343" alt="image" src="https://github.com/user-attachments/assets/53a729d5-fb15-4afb-88d7-59047543c6d6" />

<img width="1111" height="875" alt="image" src="https://github.com/user-attachments/assets/0411e3ae-5dd8-4309-9800-ade30234f516" />

----
## How this was found

This was found by running playwright axe on the page.  See [PR #551: Add Playwright tests scaffolding and initial tests](https://github.com/accessiblecommunity/accessiblecommunity.github.io/pull/551)
